src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -73,6 +73,24 @@
      * annotation and enum constant declarations.
      * Also, object blocks are children of the new keyword when defining
      * anonymous inner types.
+     * <p>
+     *     For example:
+     * </p>
+     * <pre>
+     *     class Test {}
+     * </pre>
+     * <p>
+     *     parses as:
```

Comment:
same as for `For example:` section, should be on the same line

you can use https://github.com/checkstyle/checkstyle/pull/9378/files as an example